recipes/digen/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "digen" %}
+{% set version = "0.0.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/digen-{{ version }}.tar.gz
+  sha256: ca8c7b27bd15ce4021d0ae8abce64e87be3e8717e7e40fe75ee5db0a1de79c2b
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python
```

Comment:
```suggestion
    - python >=3.6
```

Reply:
Which python version is the minimum? You tried multiple versions in the commit history.

Reply:
I would keep 3.6 as a minimal requirement.